frontend/src/pages/modelServing/screens/global/ModelServingGlobal.tsx

```diff
@@ -58,16 +60,20 @@ const ModelServingGlobal: React.FC = () => {
         emptyStatePage: <EmptyModelServing />,
       };
     }
-    if (namespace && !currentProject) {
-      renderStateProps = {
-        empty: true,
-        emptyStatePage: (
-          <InvalidProject
-            namespace={namespace}
-            getRedirectPath={(namespace) => `/modelServing/${namespace}`}
-          />
-        ),
-      };
+    if (namespace) {
+      if (!currentProject) {
```

Comment:
Why this change... you created a gap in the if-statement... if true, then false... nothing happens instead of taking preferred.

Reply:
Test this via a direct url change

Reply:
It's fine